packages/flutter_tools/lib/src/test/flutter_tester_device.dart

```diff
@@ -128,7 +127,10 @@ class FlutterTesterTestDevice extends TestDevice {
       '--enable-dart-profiling',
       '--non-interactive',
       '--use-test-fonts',
+      '--disable-asset-fonts',
```

Comment:
This is the new flag that prevents the tester from registering fonts bundled in the asset directory. It allows us to load other assets as normal while keeping the same font behavior